components/frontend/src/report/report_utils.jsx

```diff
@@ -10,8 +10,18 @@ import {
     reportPropType,
     reportsPropType,
     settingsPropType,
+    sourcePropType,
```

Comment:
Is there no `report_utils.test.jsx` ?

Reply:
No, we have 100% test coverage via the components that use the report utils. I like it that way: makes it possible to refactor those utilities without having to change unit tests.